content/applications/hr/appraisal/overview.rst

```diff
@@ -0,0 +1,123 @@
+
+.. image:: overview/appraisal-goals.png
+   :align: center
+   :alt: View all goals for each employee along with the stauts.
+
+Reporting
+=========
+
+The :guilabel:`Reporting` tab displays the :guilabel:`Appraisal Analysis` report for the company.
+The appraisal are grouped by department and can be displayed by the :guilabel:`Day`,
+:guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year`.
+
+.. image:: overview/appraisal-reporting.png
+   :align: center
+   :alt: View a color-coded chart of the appraisal status for each employee.
+
+The color of each employee's appraisal indicates the status:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Color
+     - Status
+   * - Turquoise
+     - Not Started
+   * - Peach
+     - Sent
+   * - Yellow
+     - Done
+   * - Salmon
+     - Cancelled
+
+Configuration
+=============
+
+Settings
+--------
+
+Go to :menuselection:`Appraisal --> Configuration --> Settings`, and turn on automatic appraisals by
+checking the :guilabel:`Appraisal Plans` box. Click on the :guilabel:`Appraisal Plans` internal link
+to open the :guilabel:`Appraisal Plans` section.
+
+Checking the :guilabel:`360 Feedback` box will send out pre-configured surveys. This integrates with
+the :guilabel:`Surveys` application, so :guilabel:`Surveys` must be installed in order to use this
+option. Select a :guilabel:`Default Template`, and edit the template with the
+:guilabel:`External Link` button. Edit the form in the pop up, and then click :guilabel:`Save`.
+
+Email feedback templates can be modified in the :guilabel:`Feedback Templates` section. Enter the
+text using the text editor to edit the email. Default templates are for employees and managers.
+
+.. image:: overview/appraisal-feedback-template.png
+   :align: center
+   :alt: View and edit the Feedback Templates for employees and managers.
+
+Click on the :guilabel:`Jobs Configuration` internal link to pull up a list view of all the
+:guilabel:`Job Positions` within the company. Add a new position by clicking :guilabel:`Create`,
+then enter the :guilabel:`Job Position` and add a description in the :guilabel:`Job Description`
+tab. Click on the :guilabel:`Recruitment` tab and enter all the other information for the positoin.
+The :guilabel:`Appraisals` tab will be filled in with the emails configured in the
+:guilabel:`Feedback Template` section of the :guilabel:`Settings` page. Click :guilabel:`Save` when
+done.
+
+Employees
+---------
+
+Clicking on :guilabel:`Employees` will take you to a view of all the employees in the company. This
+is an identical view to the main view in the :guilabel:`Employees` application. Any changes made
+here will be reflected on any application that uses the employees list.
+
+Appraisal Plans
+---------------
+
+This is the same view that will appear by going to :menuselection:`Settings --> Appraisal -->
+Appraisal Plans Internal Link`. Here, view how often appraisals are automatically created. The
```

Comment:
I don't see an internal link for Appraisals Plans on the settings page. Also, I wouldn't put the Appraisals header on the settings page within a menuselection label.

Reply:
Thanks- I fixed this, since I wanted to explain that there are two ways to get to the identical view. I think it's better explained now.